index.html

```diff
@@ -0,0 +1,626 @@
+                        <li>React & Vue.js</li>
+                        <li>Redux For Designers</li>
+                        <li>Vue Components</li>
+                    </ul>
+                </div>
+            </div>
+
+        </div>
+        <div class="dark-content">
+            <div class="dark-para">
+                <p class="dark-date">Guides</p>
+                <p class="dark-title">
+                    <a href="">
+                        Design Patterns & Checklists
+                    </a> </p>
+                <div class="p-content p-list">
+                    <ul>
+                        <li>Responsive Art-Direction
+                        </li>
+                        <li> Perfect Configurator
+                        </li>
+                        <li> Perfect Web Form
+                        </li>
+                        <li> Perfect Accordion
+                        </li>
+                        <li> Perfect Date Picker
+                        </li>
+                        <li> Perfect Slider
+                        </li>
+                        <li> Perfect Comparison Table
+                        </li>
+                        <li>Perfect Infinite Scroll</li>
+                        <li> Perfect Carousel
+                        </li>
+                        <li>Privacy UX</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <aside class="smashing-members smashing-news">
+            <div class="smashing-title">
+                <h1>smashing Newsletter</h1>
+            </div>
+            <div class="smashing-para">
+                <p class="news">
+                    A friendly community that keeps this website alive. With books, webinars, discounts and networking — for just 1 coffee a month.
+                    <a href="">Join the family.</a>
+                </p>
+            </div>
+            <div class="smashing-img">
+                <img src="resources/smashing.svg" alt="">
+            </div>
+            <div class="smashing-btn news-btn">
+                <button>Smashing Email</button>
+
+            </div>
+            <div class="smashing-already">
+                <p>
+                    Front-end, design and UX. Sent 2× a month.
+                </p>
+            </div>
+        </aside>
+        <footer class="footer">
+            <div class="ftitle">
+                <h2>BROWSE ALL SMASHING MAGAZINE TOPICS</h2>
+            </div>
+            <div class="foot-content">
+                <p>
+                    Accessibility Android Animation Apps CSS Design Design Patterns Design Systems E-Commerce Freebies Graphics HTML Illustrator Inspiration iOS Java Script Mobile Pattern Libraries Performance Photoshop Plugins React Responsive Web Design Service Workers
+                    Sketch Typography UI Usability User Experience Wallpapers Web Design WordPress Workflow
+                </p>
```

Comment:
- [ ] This items should be links. Kindly fix that.

Reply:
:100: